FirebaseInstallations/Source/Library/InstallationsStore/FIRInstallationsStoredItem.m

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRInstallationsStoredItem.h"
+
+@implementation FIRInstallationsStoredItem
+
+- (void)encodeWithCoder:(nonnull NSCoder *)aCoder {
+  // TODO: Implement
```

Comment:
Do you store the same info in plain file also here?

Reply:
it is used to convert `FIRInstallationsStoredItem` into `NSData` to store into the keychain. The plain file (actually GULUserDefaults) is used to store only a bool value for an appID+appName.

Reply:
Aha, that's fine, I just want to make sure it's not stored directly in plain file if it's confidential info.